### DISCUSSION TITLE
Menu locations

### DISCUSSION THREAD

**Discussion by holden** [swellstores/community#316] [Help] [answered]
Hi,

I just have a quick question. I know you guys made a bunch of changes to storefronts and there's now a `storefront_id` but where do the menus now live so I can find them in the console because when I looked at mine, it looks different from what we have on our site. It looks like you moved them out of settings? But I couldn't seem to locate them.

Thanks

eg `/settings/store/menus/header`

![CleanShot 2023-07-20 at 22 58 48](https://github.com/swellstores/community/assets/48241/5424624f-c0b4-47d9-b0ab-2486792cd935)

**Comment by steven-k1** [comment]
Hello @holden, As far as I can see, the menus are still within `/settings`

Can you provide the account ID, and I can check this to confirm?

**Comment by holden** [comment]
Sure,

```
SWELL_STORE_ID=cirqus
SWELL_STOREFRONT_ID=6kpLqOQL
```

There are items which are not in the navigation. It seems to be tied to an older storefront that I deleted.

Thanks

**Reply by steven-k1** [reply]
@holden Thanks for the details. Looking into this.

**Comment by logeshswell** [comment, marked as answer]
@holden You can find your menus here:
https://cirqus.swell.store/admin/console?get/%3Astorefronts/-6kpLqOQL~Hconfigs~Hmenus

<img width="866" alt="Screenshot 2023-07-24 at 4 47 48 PM" src="https://github.com/swellstores/community/assets/100766131/635e0dd3-4b40-49b1-9a0c-3ee94250219e">

**Reply by holden** [reply]
Perfect, thank you.
